.circleci/config.yml

```diff
@@ -55,7 +55,8 @@ jobs:
   deploy:
     <<: *defaults
     steps:
-      - checkout
+      - attach_workspace:
+          at: .
```

Comment:
Is this change intentional?

Reply:
no. not at all 👀

Reply:
This looks like it's been added by @yorinasub17 [in this commit](https://github.com/ina-stoyanova/cloud-nuke/commit/9b294286c6e55f436e18ed9064ab909a2e75ab7f)

I assume it's after a rebase on `uspstream/master, will revert anyway, as it's not related.